[2025-03-07T14:10:28.245+0100] {processor.py:186} INFO - Started process (PID=2571) to work on /Users/gabi/codes/climate_trace/climate_data_pipeline/dags/climate-data-dag.py
[2025-03-07T14:10:28.249+0100] {processor.py:914} INFO - Processing file /Users/gabi/codes/climate_trace/climate_data_pipeline/dags/climate-data-dag.py for tasks to queue
[2025-03-07T14:10:28.250+0100] {logging_mixin.py:190} INFO - [2025-03-07T14:10:28.249+0100] {dagbag.py:588} INFO - Filling up the DagBag from /Users/gabi/codes/climate_trace/climate_data_pipeline/dags/climate-data-dag.py
[2025-03-07T14:10:29.163+0100] {processor.py:925} INFO - DAG(s) 'climate_data_pipeline' retrieved from /Users/gabi/codes/climate_trace/climate_data_pipeline/dags/climate-data-dag.py
[2025-03-07T14:10:29.167+0100] {logging_mixin.py:190} INFO - [2025-03-07T14:10:29.167+0100] {dag.py:3239} INFO - Sync 1 DAGs
[2025-03-07T14:10:29.240+0100] {logging_mixin.py:190} INFO - [2025-03-07T14:10:29.240+0100] {dag.py:4180} INFO - Setting next_dagrun for climate_data_pipeline to 2025-03-06T00:00:00+00:00, run_after=2025-03-07T00:00:00+00:00
[2025-03-07T14:10:29.253+0100] {processor.py:208} INFO - Processing /Users/gabi/codes/climate_trace/climate_data_pipeline/dags/climate-data-dag.py took 1.011 seconds
[2025-03-07T14:13:19.102+0100] {processor.py:186} INFO - Started process (PID=2919) to work on /Users/gabi/codes/climate_trace/climate_data_pipeline/dags/climate-data-dag.py
[2025-03-07T14:13:19.105+0100] {processor.py:914} INFO - Processing file /Users/gabi/codes/climate_trace/climate_data_pipeline/dags/climate-data-dag.py for tasks to queue
[2025-03-07T14:13:19.106+0100] {logging_mixin.py:190} INFO - [2025-03-07T14:13:19.106+0100] {dagbag.py:588} INFO - Filling up the DagBag from /Users/gabi/codes/climate_trace/climate_data_pipeline/dags/climate-data-dag.py
[2025-03-07T14:13:20.007+0100] {processor.py:925} INFO - DAG(s) 'climate_data_pipeline' retrieved from /Users/gabi/codes/climate_trace/climate_data_pipeline/dags/climate-data-dag.py
[2025-03-07T14:13:20.132+0100] {logging_mixin.py:190} INFO - [2025-03-07T14:13:20.132+0100] {dag.py:3239} INFO - Sync 1 DAGs
[2025-03-07T14:13:20.143+0100] {logging_mixin.py:190} INFO - [2025-03-07T14:13:20.143+0100] {dag.py:4180} INFO - Setting next_dagrun for climate_data_pipeline to 2025-03-06T00:00:00+00:00, run_after=2025-03-07T00:00:00+00:00
[2025-03-07T14:13:20.155+0100] {processor.py:208} INFO - Processing /Users/gabi/codes/climate_trace/climate_data_pipeline/dags/climate-data-dag.py took 1.056 seconds
[2025-03-07T14:16:08.447+0100] {processor.py:186} INFO - Started process (PID=3194) to work on /Users/gabi/codes/climate_trace/climate_data_pipeline/dags/climate-data-dag.py
[2025-03-07T14:16:08.450+0100] {processor.py:914} INFO - Processing file /Users/gabi/codes/climate_trace/climate_data_pipeline/dags/climate-data-dag.py for tasks to queue
[2025-03-07T14:16:08.451+0100] {logging_mixin.py:190} INFO - [2025-03-07T14:16:08.451+0100] {dagbag.py:588} INFO - Filling up the DagBag from /Users/gabi/codes/climate_trace/climate_data_pipeline/dags/climate-data-dag.py
[2025-03-07T14:16:09.341+0100] {processor.py:925} INFO - DAG(s) 'climate_data_pipeline' retrieved from /Users/gabi/codes/climate_trace/climate_data_pipeline/dags/climate-data-dag.py
[2025-03-07T14:16:09.345+0100] {logging_mixin.py:190} INFO - [2025-03-07T14:16:09.345+0100] {dag.py:3239} INFO - Sync 1 DAGs
[2025-03-07T14:16:09.415+0100] {logging_mixin.py:190} INFO - [2025-03-07T14:16:09.415+0100] {dag.py:4180} INFO - Setting next_dagrun for climate_data_pipeline to 2025-03-07T00:00:00+00:00, run_after=2025-03-08T00:00:00+00:00
[2025-03-07T14:16:09.426+0100] {processor.py:208} INFO - Processing /Users/gabi/codes/climate_trace/climate_data_pipeline/dags/climate-data-dag.py took 0.982 seconds
[2025-03-07T14:18:22.455+0100] {processor.py:186} INFO - Started process (PID=3443) to work on /Users/gabi/codes/climate_trace/climate_data_pipeline/dags/climate-data-dag.py
[2025-03-07T14:18:22.458+0100] {processor.py:914} INFO - Processing file /Users/gabi/codes/climate_trace/climate_data_pipeline/dags/climate-data-dag.py for tasks to queue
[2025-03-07T14:18:22.459+0100] {logging_mixin.py:190} INFO - [2025-03-07T14:18:22.459+0100] {dagbag.py:588} INFO - Filling up the DagBag from /Users/gabi/codes/climate_trace/climate_data_pipeline/dags/climate-data-dag.py
[2025-03-07T14:18:23.326+0100] {processor.py:925} INFO - DAG(s) 'climate_data_pipeline' retrieved from /Users/gabi/codes/climate_trace/climate_data_pipeline/dags/climate-data-dag.py
[2025-03-07T14:18:23.444+0100] {logging_mixin.py:190} INFO - [2025-03-07T14:18:23.444+0100] {dag.py:3239} INFO - Sync 1 DAGs
[2025-03-07T14:18:23.455+0100] {logging_mixin.py:190} INFO - [2025-03-07T14:18:23.455+0100] {dag.py:4180} INFO - Setting next_dagrun for climate_data_pipeline to 2025-03-07T00:00:00+00:00, run_after=2025-03-08T00:00:00+00:00
[2025-03-07T14:18:23.466+0100] {processor.py:208} INFO - Processing /Users/gabi/codes/climate_trace/climate_data_pipeline/dags/climate-data-dag.py took 1.014 seconds
[2025-03-07T14:20:37.372+0100] {processor.py:186} INFO - Started process (PID=3744) to work on /Users/gabi/codes/climate_trace/climate_data_pipeline/dags/climate-data-dag.py
[2025-03-07T14:20:37.376+0100] {processor.py:914} INFO - Processing file /Users/gabi/codes/climate_trace/climate_data_pipeline/dags/climate-data-dag.py for tasks to queue
[2025-03-07T14:20:37.377+0100] {logging_mixin.py:190} INFO - [2025-03-07T14:20:37.377+0100] {dagbag.py:588} INFO - Filling up the DagBag from /Users/gabi/codes/climate_trace/climate_data_pipeline/dags/climate-data-dag.py
[2025-03-07T14:20:38.243+0100] {processor.py:925} INFO - DAG(s) 'climate_data_pipeline' retrieved from /Users/gabi/codes/climate_trace/climate_data_pipeline/dags/climate-data-dag.py
[2025-03-07T14:20:38.247+0100] {logging_mixin.py:190} INFO - [2025-03-07T14:20:38.247+0100] {dag.py:3239} INFO - Sync 1 DAGs
[2025-03-07T14:20:38.314+0100] {logging_mixin.py:190} INFO - [2025-03-07T14:20:38.314+0100] {dag.py:4180} INFO - Setting next_dagrun for climate_data_pipeline to 2025-03-07T00:00:00+00:00, run_after=2025-03-08T00:00:00+00:00
[2025-03-07T14:20:38.326+0100] {processor.py:208} INFO - Processing /Users/gabi/codes/climate_trace/climate_data_pipeline/dags/climate-data-dag.py took 0.956 seconds
[2025-03-07T14:22:49.414+0100] {processor.py:186} INFO - Started process (PID=3968) to work on /Users/gabi/codes/climate_trace/climate_data_pipeline/dags/climate-data-dag.py
[2025-03-07T14:22:49.417+0100] {processor.py:914} INFO - Processing file /Users/gabi/codes/climate_trace/climate_data_pipeline/dags/climate-data-dag.py for tasks to queue
[2025-03-07T14:22:49.418+0100] {logging_mixin.py:190} INFO - [2025-03-07T14:22:49.418+0100] {dagbag.py:588} INFO - Filling up the DagBag from /Users/gabi/codes/climate_trace/climate_data_pipeline/dags/climate-data-dag.py
[2025-03-07T14:22:50.300+0100] {logging_mixin.py:190} INFO - [2025-03-07T14:22:50.299+0100] {dagbag.py:387} ERROR - Failed to import: /Users/gabi/codes/climate_trace/climate_data_pipeline/dags/climate-data-dag.py
Traceback (most recent call last):
  File "/usr/local/anaconda3/envs/airflow_env/lib/python3.8/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/Users/gabi/codes/climate_trace/climate_data_pipeline/dags/climate-data-dag.py", line 11, in <module>
    from dlt_extract import run_world_bank_pipeline, run_climate_trace_pipeline
  File "/Users/gabi/codes/climate_trace/climate_data_pipeline/scripts/dlt_extract.py", line 1, in <module>
    import dlt
ModuleNotFoundError: No module named 'dlt'
[2025-03-07T14:22:50.300+0100] {processor.py:927} WARNING - No viable dags retrieved from /Users/gabi/codes/climate_trace/climate_data_pipeline/dags/climate-data-dag.py
[2025-03-07T14:22:50.378+0100] {processor.py:208} INFO - Processing /Users/gabi/codes/climate_trace/climate_data_pipeline/dags/climate-data-dag.py took 0.967 seconds
[2025-03-07T14:25:12.082+0100] {processor.py:186} INFO - Started process (PID=4191) to work on /Users/gabi/codes/climate_trace/climate_data_pipeline/dags/climate-data-dag.py
[2025-03-07T14:25:12.085+0100] {processor.py:914} INFO - Processing file /Users/gabi/codes/climate_trace/climate_data_pipeline/dags/climate-data-dag.py for tasks to queue
[2025-03-07T14:25:12.085+0100] {logging_mixin.py:190} INFO - [2025-03-07T14:25:12.085+0100] {dagbag.py:588} INFO - Filling up the DagBag from /Users/gabi/codes/climate_trace/climate_data_pipeline/dags/climate-data-dag.py
[2025-03-07T14:25:12.971+0100] {logging_mixin.py:190} INFO - [2025-03-07T14:25:12.970+0100] {dagbag.py:387} ERROR - Failed to import: /Users/gabi/codes/climate_trace/climate_data_pipeline/dags/climate-data-dag.py
Traceback (most recent call last):
  File "/usr/local/anaconda3/envs/airflow_env/lib/python3.8/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/Users/gabi/codes/climate_trace/climate_data_pipeline/dags/climate-data-dag.py", line 11, in <module>
    from dlt_extract import run_world_bank_pipeline, run_climate_trace_pipeline
  File "/Users/gabi/codes/climate_trace/climate_data_pipeline/scripts/dlt_extract.py", line 1, in <module>
    import dlt
ModuleNotFoundError: No module named 'dlt'
[2025-03-07T14:25:12.971+0100] {processor.py:927} WARNING - No viable dags retrieved from /Users/gabi/codes/climate_trace/climate_data_pipeline/dags/climate-data-dag.py
[2025-03-07T14:25:13.054+0100] {processor.py:208} INFO - Processing /Users/gabi/codes/climate_trace/climate_data_pipeline/dags/climate-data-dag.py took 0.975 seconds
[2025-03-07T14:26:34.626+0100] {processor.py:186} INFO - Started process (PID=4326) to work on /Users/gabi/codes/climate_trace/climate_data_pipeline/dags/climate-data-dag.py
[2025-03-07T14:26:34.629+0100] {processor.py:914} INFO - Processing file /Users/gabi/codes/climate_trace/climate_data_pipeline/dags/climate-data-dag.py for tasks to queue
[2025-03-07T14:26:34.630+0100] {logging_mixin.py:190} INFO - [2025-03-07T14:26:34.630+0100] {dagbag.py:588} INFO - Filling up the DagBag from /Users/gabi/codes/climate_trace/climate_data_pipeline/dags/climate-data-dag.py
[2025-03-07T14:26:35.549+0100] {logging_mixin.py:190} INFO - [2025-03-07T14:26:35.548+0100] {dagbag.py:387} ERROR - Failed to import: /Users/gabi/codes/climate_trace/climate_data_pipeline/dags/climate-data-dag.py
Traceback (most recent call last):
  File "/usr/local/anaconda3/envs/airflow_env/lib/python3.8/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/Users/gabi/codes/climate_trace/climate_data_pipeline/dags/climate-data-dag.py", line 11, in <module>
    from dlt_extract import run_world_bank_pipeline, run_climate_trace_pipeline
  File "/Users/gabi/codes/climate_trace/climate_data_pipeline/scripts/dlt_extract.py", line 1, in <module>
    import dlt
ModuleNotFoundError: No module named 'dlt'
[2025-03-07T14:26:35.550+0100] {processor.py:927} WARNING - No viable dags retrieved from /Users/gabi/codes/climate_trace/climate_data_pipeline/dags/climate-data-dag.py
[2025-03-07T14:26:35.635+0100] {processor.py:208} INFO - Processing /Users/gabi/codes/climate_trace/climate_data_pipeline/dags/climate-data-dag.py took 1.012 seconds
[2025-03-07T14:43:18.519+0100] {processor.py:186} INFO - Started process (PID=4835) to work on /Users/gabi/codes/climate_trace/climate_data_pipeline/dags/climate-data-dag.py
[2025-03-07T14:43:18.522+0100] {processor.py:914} INFO - Processing file /Users/gabi/codes/climate_trace/climate_data_pipeline/dags/climate-data-dag.py for tasks to queue
[2025-03-07T14:43:18.523+0100] {logging_mixin.py:190} INFO - [2025-03-07T14:43:18.523+0100] {dagbag.py:588} INFO - Filling up the DagBag from /Users/gabi/codes/climate_trace/climate_data_pipeline/dags/climate-data-dag.py
[2025-03-07T13:43:20.047+0000] {processor.py:925} INFO - DAG(s) 'climate_data_pipeline' retrieved from /Users/gabi/codes/climate_trace/climate_data_pipeline/dags/climate-data-dag.py
[2025-03-07T13:43:20.221+0000] {logging_mixin.py:190} INFO - [2025-03-07T13:43:20.221+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-03-07T13:43:20.233+0000] {logging_mixin.py:190} INFO - [2025-03-07T13:43:20.232+0000] {dag.py:4180} INFO - Setting next_dagrun for climate_data_pipeline to 2025-03-07T00:00:00+00:00, run_after=2025-03-08T00:00:00+00:00
[2025-03-07T13:43:20.248+0000] {processor.py:208} INFO - Processing /Users/gabi/codes/climate_trace/climate_data_pipeline/dags/climate-data-dag.py took 1.733 seconds
[2025-03-07T14:46:05.866+0100] {processor.py:186} INFO - Started process (PID=5064) to work on /Users/gabi/codes/climate_trace/climate_data_pipeline/dags/climate-data-dag.py
[2025-03-07T14:46:05.869+0100] {processor.py:914} INFO - Processing file /Users/gabi/codes/climate_trace/climate_data_pipeline/dags/climate-data-dag.py for tasks to queue
[2025-03-07T14:46:05.870+0100] {logging_mixin.py:190} INFO - [2025-03-07T14:46:05.870+0100] {dagbag.py:588} INFO - Filling up the DagBag from /Users/gabi/codes/climate_trace/climate_data_pipeline/dags/climate-data-dag.py
[2025-03-07T13:46:07.409+0000] {processor.py:925} INFO - DAG(s) 'climate_data_pipeline' retrieved from /Users/gabi/codes/climate_trace/climate_data_pipeline/dags/climate-data-dag.py
[2025-03-07T13:46:07.544+0000] {logging_mixin.py:190} INFO - [2025-03-07T13:46:07.544+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-03-07T13:46:07.591+0000] {logging_mixin.py:190} INFO - [2025-03-07T13:46:07.591+0000] {dag.py:4180} INFO - Setting next_dagrun for climate_data_pipeline to 2025-03-07T00:00:00+00:00, run_after=2025-03-08T00:00:00+00:00
[2025-03-07T13:46:07.618+0000] {processor.py:208} INFO - Processing /Users/gabi/codes/climate_trace/climate_data_pipeline/dags/climate-data-dag.py took 1.754 seconds
[2025-03-07T14:48:25.693+0100] {processor.py:186} INFO - Started process (PID=5327) to work on /Users/gabi/codes/climate_trace/climate_data_pipeline/dags/climate-data-dag.py
[2025-03-07T14:48:25.697+0100] {processor.py:914} INFO - Processing file /Users/gabi/codes/climate_trace/climate_data_pipeline/dags/climate-data-dag.py for tasks to queue
[2025-03-07T14:48:25.697+0100] {logging_mixin.py:190} INFO - [2025-03-07T14:48:25.697+0100] {dagbag.py:588} INFO - Filling up the DagBag from /Users/gabi/codes/climate_trace/climate_data_pipeline/dags/climate-data-dag.py
[2025-03-07T13:48:27.136+0000] {processor.py:925} INFO - DAG(s) 'climate_data_pipeline' retrieved from /Users/gabi/codes/climate_trace/climate_data_pipeline/dags/climate-data-dag.py
[2025-03-07T13:48:27.248+0000] {logging_mixin.py:190} INFO - [2025-03-07T13:48:27.247+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-03-07T13:48:27.259+0000] {logging_mixin.py:190} INFO - [2025-03-07T13:48:27.259+0000] {dag.py:4180} INFO - Setting next_dagrun for climate_data_pipeline to 2025-03-07T00:00:00+00:00, run_after=2025-03-08T00:00:00+00:00
[2025-03-07T13:48:27.270+0000] {processor.py:208} INFO - Processing /Users/gabi/codes/climate_trace/climate_data_pipeline/dags/climate-data-dag.py took 1.580 seconds
[2025-03-07T14:49:12.593+0100] {processor.py:186} INFO - Started process (PID=5452) to work on /Users/gabi/codes/climate_trace/climate_data_pipeline/dags/climate-data-dag.py
[2025-03-07T14:49:12.596+0100] {processor.py:914} INFO - Processing file /Users/gabi/codes/climate_trace/climate_data_pipeline/dags/climate-data-dag.py for tasks to queue
[2025-03-07T14:49:12.597+0100] {logging_mixin.py:190} INFO - [2025-03-07T14:49:12.597+0100] {dagbag.py:588} INFO - Filling up the DagBag from /Users/gabi/codes/climate_trace/climate_data_pipeline/dags/climate-data-dag.py
[2025-03-07T13:49:14.014+0000] {processor.py:925} INFO - DAG(s) 'climate_data_pipeline' retrieved from /Users/gabi/codes/climate_trace/climate_data_pipeline/dags/climate-data-dag.py
[2025-03-07T13:49:14.134+0000] {logging_mixin.py:190} INFO - [2025-03-07T13:49:14.133+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-03-07T13:49:14.147+0000] {logging_mixin.py:190} INFO - [2025-03-07T13:49:14.146+0000] {dag.py:4180} INFO - Setting next_dagrun for climate_data_pipeline to 2025-03-07T00:00:00+00:00, run_after=2025-03-08T00:00:00+00:00
[2025-03-07T13:49:14.159+0000] {processor.py:208} INFO - Processing /Users/gabi/codes/climate_trace/climate_data_pipeline/dags/climate-data-dag.py took 1.569 seconds
[2025-03-07T14:51:44.983+0100] {processor.py:186} INFO - Started process (PID=5795) to work on /Users/gabi/codes/climate_trace/climate_data_pipeline/dags/climate-data-dag.py
[2025-03-07T14:51:44.986+0100] {processor.py:914} INFO - Processing file /Users/gabi/codes/climate_trace/climate_data_pipeline/dags/climate-data-dag.py for tasks to queue
[2025-03-07T14:51:44.987+0100] {logging_mixin.py:190} INFO - [2025-03-07T14:51:44.987+0100] {dagbag.py:588} INFO - Filling up the DagBag from /Users/gabi/codes/climate_trace/climate_data_pipeline/dags/climate-data-dag.py
[2025-03-07T14:51:45.855+0100] {logging_mixin.py:190} INFO - [2025-03-07T14:51:45.855+0100] {dagbag.py:387} ERROR - Failed to import: /Users/gabi/codes/climate_trace/climate_data_pipeline/dags/climate-data-dag.py
Traceback (most recent call last):
  File "/usr/local/anaconda3/envs/airflow_env/lib/python3.8/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/Users/gabi/codes/climate_trace/climate_data_pipeline/dags/climate-data-dag.py", line 11, in <module>
    from dlt_extract import run_world_bank_pipeline, run_climate_trace_pipeline
ModuleNotFoundError: No module named 'dlt_extract'
[2025-03-07T14:51:45.856+0100] {processor.py:927} WARNING - No viable dags retrieved from /Users/gabi/codes/climate_trace/climate_data_pipeline/dags/climate-data-dag.py
[2025-03-07T14:51:45.933+0100] {processor.py:208} INFO - Processing /Users/gabi/codes/climate_trace/climate_data_pipeline/dags/climate-data-dag.py took 0.953 seconds
[2025-03-07T14:52:37.820+0100] {processor.py:186} INFO - Started process (PID=5916) to work on /Users/gabi/codes/climate_trace/climate_data_pipeline/dags/climate-data-dag.py
[2025-03-07T14:52:37.823+0100] {processor.py:914} INFO - Processing file /Users/gabi/codes/climate_trace/climate_data_pipeline/dags/climate-data-dag.py for tasks to queue
[2025-03-07T14:52:37.824+0100] {logging_mixin.py:190} INFO - [2025-03-07T14:52:37.824+0100] {dagbag.py:588} INFO - Filling up the DagBag from /Users/gabi/codes/climate_trace/climate_data_pipeline/dags/climate-data-dag.py
[2025-03-07T13:52:39.236+0000] {processor.py:925} INFO - DAG(s) 'climate_data_pipeline' retrieved from /Users/gabi/codes/climate_trace/climate_data_pipeline/dags/climate-data-dag.py
[2025-03-07T13:52:39.350+0000] {logging_mixin.py:190} INFO - [2025-03-07T13:52:39.350+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-03-07T13:52:39.362+0000] {logging_mixin.py:190} INFO - [2025-03-07T13:52:39.362+0000] {dag.py:4180} INFO - Setting next_dagrun for climate_data_pipeline to 2025-03-07T00:00:00+00:00, run_after=2025-03-08T00:00:00+00:00
[2025-03-07T13:52:39.375+0000] {processor.py:208} INFO - Processing /Users/gabi/codes/climate_trace/climate_data_pipeline/dags/climate-data-dag.py took 1.558 seconds
[2025-03-07T14:55:25.385+0100] {processor.py:186} INFO - Started process (PID=6226) to work on /Users/gabi/codes/climate_trace/climate_data_pipeline/dags/climate-data-dag.py
[2025-03-07T14:55:25.388+0100] {processor.py:914} INFO - Processing file /Users/gabi/codes/climate_trace/climate_data_pipeline/dags/climate-data-dag.py for tasks to queue
[2025-03-07T14:55:25.389+0100] {logging_mixin.py:190} INFO - [2025-03-07T14:55:25.389+0100] {dagbag.py:588} INFO - Filling up the DagBag from /Users/gabi/codes/climate_trace/climate_data_pipeline/dags/climate-data-dag.py
[2025-03-07T13:55:26.829+0000] {processor.py:925} INFO - DAG(s) 'climate_data_pipeline' retrieved from /Users/gabi/codes/climate_trace/climate_data_pipeline/dags/climate-data-dag.py
[2025-03-07T13:55:26.991+0000] {logging_mixin.py:190} INFO - [2025-03-07T13:55:26.990+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-03-07T13:55:27.002+0000] {logging_mixin.py:190} INFO - [2025-03-07T13:55:27.001+0000] {dag.py:4180} INFO - Setting next_dagrun for climate_data_pipeline to 2025-03-07T00:00:00+00:00, run_after=2025-03-08T00:00:00+00:00
[2025-03-07T13:55:27.015+0000] {processor.py:208} INFO - Processing /Users/gabi/codes/climate_trace/climate_data_pipeline/dags/climate-data-dag.py took 1.633 seconds
[2025-03-07T14:57:37.657+0100] {processor.py:186} INFO - Started process (PID=6495) to work on /Users/gabi/codes/climate_trace/climate_data_pipeline/dags/climate-data-dag.py
[2025-03-07T14:57:37.660+0100] {processor.py:914} INFO - Processing file /Users/gabi/codes/climate_trace/climate_data_pipeline/dags/climate-data-dag.py for tasks to queue
[2025-03-07T14:57:37.661+0100] {logging_mixin.py:190} INFO - [2025-03-07T14:57:37.661+0100] {dagbag.py:588} INFO - Filling up the DagBag from /Users/gabi/codes/climate_trace/climate_data_pipeline/dags/climate-data-dag.py
[2025-03-07T13:57:39.061+0000] {processor.py:925} INFO - DAG(s) 'climate_data_pipeline' retrieved from /Users/gabi/codes/climate_trace/climate_data_pipeline/dags/climate-data-dag.py
[2025-03-07T13:57:39.170+0000] {logging_mixin.py:190} INFO - [2025-03-07T13:57:39.170+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-03-07T13:57:39.182+0000] {logging_mixin.py:190} INFO - [2025-03-07T13:57:39.181+0000] {dag.py:4180} INFO - Setting next_dagrun for climate_data_pipeline to 2025-03-07T00:00:00+00:00, run_after=2025-03-08T00:00:00+00:00
[2025-03-07T13:57:39.193+0000] {processor.py:208} INFO - Processing /Users/gabi/codes/climate_trace/climate_data_pipeline/dags/climate-data-dag.py took 1.539 seconds
[2025-03-07T14:59:47.011+0100] {processor.py:186} INFO - Started process (PID=6777) to work on /Users/gabi/codes/climate_trace/climate_data_pipeline/dags/climate-data-dag.py
[2025-03-07T14:59:47.014+0100] {processor.py:914} INFO - Processing file /Users/gabi/codes/climate_trace/climate_data_pipeline/dags/climate-data-dag.py for tasks to queue
[2025-03-07T14:59:47.015+0100] {logging_mixin.py:190} INFO - [2025-03-07T14:59:47.015+0100] {dagbag.py:588} INFO - Filling up the DagBag from /Users/gabi/codes/climate_trace/climate_data_pipeline/dags/climate-data-dag.py
[2025-03-07T13:59:48.446+0000] {processor.py:925} INFO - DAG(s) 'climate_data_pipeline' retrieved from /Users/gabi/codes/climate_trace/climate_data_pipeline/dags/climate-data-dag.py
[2025-03-07T13:59:48.559+0000] {logging_mixin.py:190} INFO - [2025-03-07T13:59:48.559+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-03-07T13:59:48.571+0000] {logging_mixin.py:190} INFO - [2025-03-07T13:59:48.571+0000] {dag.py:4180} INFO - Setting next_dagrun for climate_data_pipeline to 2025-03-07T00:00:00+00:00, run_after=2025-03-08T00:00:00+00:00
[2025-03-07T13:59:48.582+0000] {processor.py:208} INFO - Processing /Users/gabi/codes/climate_trace/climate_data_pipeline/dags/climate-data-dag.py took 1.574 seconds
[2025-03-07T15:02:17.691+0100] {processor.py:186} INFO - Started process (PID=7043) to work on /Users/gabi/codes/climate_trace/climate_data_pipeline/dags/climate-data-dag.py
[2025-03-07T15:02:17.695+0100] {processor.py:914} INFO - Processing file /Users/gabi/codes/climate_trace/climate_data_pipeline/dags/climate-data-dag.py for tasks to queue
[2025-03-07T15:02:17.695+0100] {logging_mixin.py:190} INFO - [2025-03-07T15:02:17.695+0100] {dagbag.py:588} INFO - Filling up the DagBag from /Users/gabi/codes/climate_trace/climate_data_pipeline/dags/climate-data-dag.py
[2025-03-07T14:02:19.170+0000] {processor.py:925} INFO - DAG(s) 'climate_data_pipeline' retrieved from /Users/gabi/codes/climate_trace/climate_data_pipeline/dags/climate-data-dag.py
[2025-03-07T14:02:19.286+0000] {logging_mixin.py:190} INFO - [2025-03-07T14:02:19.286+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-03-07T14:02:19.297+0000] {logging_mixin.py:190} INFO - [2025-03-07T14:02:19.297+0000] {dag.py:4180} INFO - Setting next_dagrun for climate_data_pipeline to 2025-03-07T00:00:00+00:00, run_after=2025-03-08T00:00:00+00:00
[2025-03-07T14:02:19.309+0000] {processor.py:208} INFO - Processing /Users/gabi/codes/climate_trace/climate_data_pipeline/dags/climate-data-dag.py took 1.621 seconds
[2025-03-07T15:04:39.718+0100] {processor.py:186} INFO - Started process (PID=7250) to work on /Users/gabi/codes/climate_trace/climate_data_pipeline/dags/climate-data-dag.py
[2025-03-07T15:04:39.721+0100] {processor.py:914} INFO - Processing file /Users/gabi/codes/climate_trace/climate_data_pipeline/dags/climate-data-dag.py for tasks to queue
[2025-03-07T15:04:39.722+0100] {logging_mixin.py:190} INFO - [2025-03-07T15:04:39.722+0100] {dagbag.py:588} INFO - Filling up the DagBag from /Users/gabi/codes/climate_trace/climate_data_pipeline/dags/climate-data-dag.py
[2025-03-07T14:04:41.175+0000] {processor.py:925} INFO - DAG(s) 'climate_data_pipeline' retrieved from /Users/gabi/codes/climate_trace/climate_data_pipeline/dags/climate-data-dag.py
[2025-03-07T14:04:41.293+0000] {logging_mixin.py:190} INFO - [2025-03-07T14:04:41.293+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-03-07T14:04:41.304+0000] {logging_mixin.py:190} INFO - [2025-03-07T14:04:41.304+0000] {dag.py:4180} INFO - Setting next_dagrun for climate_data_pipeline to 2025-03-07T00:00:00+00:00, run_after=2025-03-08T00:00:00+00:00
[2025-03-07T14:04:41.316+0000] {processor.py:208} INFO - Processing /Users/gabi/codes/climate_trace/climate_data_pipeline/dags/climate-data-dag.py took 1.601 seconds
[2025-03-07T15:07:02.920+0100] {processor.py:186} INFO - Started process (PID=7542) to work on /Users/gabi/codes/climate_trace/climate_data_pipeline/dags/climate-data-dag.py
[2025-03-07T15:07:02.923+0100] {processor.py:914} INFO - Processing file /Users/gabi/codes/climate_trace/climate_data_pipeline/dags/climate-data-dag.py for tasks to queue
[2025-03-07T15:07:02.924+0100] {logging_mixin.py:190} INFO - [2025-03-07T15:07:02.924+0100] {dagbag.py:588} INFO - Filling up the DagBag from /Users/gabi/codes/climate_trace/climate_data_pipeline/dags/climate-data-dag.py
[2025-03-07T14:07:04.381+0000] {processor.py:925} INFO - DAG(s) 'climate_data_pipeline' retrieved from /Users/gabi/codes/climate_trace/climate_data_pipeline/dags/climate-data-dag.py
[2025-03-07T14:07:04.496+0000] {logging_mixin.py:190} INFO - [2025-03-07T14:07:04.496+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-03-07T14:07:04.507+0000] {logging_mixin.py:190} INFO - [2025-03-07T14:07:04.507+0000] {dag.py:4180} INFO - Setting next_dagrun for climate_data_pipeline to 2025-03-07T00:00:00+00:00, run_after=2025-03-08T00:00:00+00:00
[2025-03-07T14:07:04.519+0000] {processor.py:208} INFO - Processing /Users/gabi/codes/climate_trace/climate_data_pipeline/dags/climate-data-dag.py took 1.602 seconds
[2025-03-07T15:09:29.740+0100] {processor.py:186} INFO - Started process (PID=7793) to work on /Users/gabi/codes/climate_trace/climate_data_pipeline/dags/climate-data-dag.py
[2025-03-07T15:09:29.744+0100] {processor.py:914} INFO - Processing file /Users/gabi/codes/climate_trace/climate_data_pipeline/dags/climate-data-dag.py for tasks to queue
[2025-03-07T15:09:29.745+0100] {logging_mixin.py:190} INFO - [2025-03-07T15:09:29.745+0100] {dagbag.py:588} INFO - Filling up the DagBag from /Users/gabi/codes/climate_trace/climate_data_pipeline/dags/climate-data-dag.py
[2025-03-07T14:09:31.226+0000] {processor.py:925} INFO - DAG(s) 'climate_data_pipeline' retrieved from /Users/gabi/codes/climate_trace/climate_data_pipeline/dags/climate-data-dag.py
[2025-03-07T14:09:31.343+0000] {logging_mixin.py:190} INFO - [2025-03-07T14:09:31.343+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-03-07T14:09:31.355+0000] {logging_mixin.py:190} INFO - [2025-03-07T14:09:31.355+0000] {dag.py:4180} INFO - Setting next_dagrun for climate_data_pipeline to 2025-03-07T00:00:00+00:00, run_after=2025-03-08T00:00:00+00:00
[2025-03-07T14:09:31.367+0000] {processor.py:208} INFO - Processing /Users/gabi/codes/climate_trace/climate_data_pipeline/dags/climate-data-dag.py took 1.629 seconds
[2025-03-07T15:12:04.021+0100] {processor.py:186} INFO - Started process (PID=8080) to work on /Users/gabi/codes/climate_trace/climate_data_pipeline/dags/climate-data-dag.py
[2025-03-07T15:12:04.024+0100] {processor.py:914} INFO - Processing file /Users/gabi/codes/climate_trace/climate_data_pipeline/dags/climate-data-dag.py for tasks to queue
[2025-03-07T15:12:04.025+0100] {logging_mixin.py:190} INFO - [2025-03-07T15:12:04.025+0100] {dagbag.py:588} INFO - Filling up the DagBag from /Users/gabi/codes/climate_trace/climate_data_pipeline/dags/climate-data-dag.py
[2025-03-07T14:12:05.436+0000] {processor.py:925} INFO - DAG(s) 'climate_data_pipeline' retrieved from /Users/gabi/codes/climate_trace/climate_data_pipeline/dags/climate-data-dag.py
[2025-03-07T14:12:05.561+0000] {logging_mixin.py:190} INFO - [2025-03-07T14:12:05.561+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-03-07T14:12:05.574+0000] {logging_mixin.py:190} INFO - [2025-03-07T14:12:05.574+0000] {dag.py:4180} INFO - Setting next_dagrun for climate_data_pipeline to 2025-03-07T00:00:00+00:00, run_after=2025-03-08T00:00:00+00:00
[2025-03-07T14:12:05.588+0000] {processor.py:208} INFO - Processing /Users/gabi/codes/climate_trace/climate_data_pipeline/dags/climate-data-dag.py took 1.570 seconds
[2025-03-07T15:15:08.477+0100] {processor.py:186} INFO - Started process (PID=8439) to work on /Users/gabi/codes/climate_trace/climate_data_pipeline/dags/climate-data-dag.py
[2025-03-07T15:15:08.480+0100] {processor.py:914} INFO - Processing file /Users/gabi/codes/climate_trace/climate_data_pipeline/dags/climate-data-dag.py for tasks to queue
[2025-03-07T15:15:08.481+0100] {logging_mixin.py:190} INFO - [2025-03-07T15:15:08.481+0100] {dagbag.py:588} INFO - Filling up the DagBag from /Users/gabi/codes/climate_trace/climate_data_pipeline/dags/climate-data-dag.py
[2025-03-07T14:15:09.890+0000] {processor.py:925} INFO - DAG(s) 'climate_data_pipeline' retrieved from /Users/gabi/codes/climate_trace/climate_data_pipeline/dags/climate-data-dag.py
[2025-03-07T14:15:10.002+0000] {logging_mixin.py:190} INFO - [2025-03-07T14:15:10.002+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-03-07T14:15:10.014+0000] {logging_mixin.py:190} INFO - [2025-03-07T14:15:10.013+0000] {dag.py:4180} INFO - Setting next_dagrun for climate_data_pipeline to 2025-03-07T00:00:00+00:00, run_after=2025-03-08T00:00:00+00:00
[2025-03-07T14:15:10.025+0000] {processor.py:208} INFO - Processing /Users/gabi/codes/climate_trace/climate_data_pipeline/dags/climate-data-dag.py took 1.551 seconds
[2025-03-07T15:17:29.766+0100] {processor.py:186} INFO - Started process (PID=8651) to work on /Users/gabi/codes/climate_trace/climate_data_pipeline/dags/climate-data-dag.py
[2025-03-07T15:17:29.769+0100] {processor.py:914} INFO - Processing file /Users/gabi/codes/climate_trace/climate_data_pipeline/dags/climate-data-dag.py for tasks to queue
[2025-03-07T15:17:29.770+0100] {logging_mixin.py:190} INFO - [2025-03-07T15:17:29.770+0100] {dagbag.py:588} INFO - Filling up the DagBag from /Users/gabi/codes/climate_trace/climate_data_pipeline/dags/climate-data-dag.py
[2025-03-07T15:17:30.655+0100] {logging_mixin.py:190} INFO - [2025-03-07T15:17:30.654+0100] {dagbag.py:387} ERROR - Failed to import: /Users/gabi/codes/climate_trace/climate_data_pipeline/dags/climate-data-dag.py
Traceback (most recent call last):
  File "/usr/local/anaconda3/envs/airflow_env/lib/python3.8/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/Users/gabi/codes/climate_trace/climate_data_pipeline/dags/climate-data-dag.py", line 11, in <module>
    from dlt_extractor import run_world_bank_pipeline, run_climate_trace_pipeline
ModuleNotFoundError: No module named 'dlt_extractor'
[2025-03-07T15:17:30.655+0100] {processor.py:927} WARNING - No viable dags retrieved from /Users/gabi/codes/climate_trace/climate_data_pipeline/dags/climate-data-dag.py
[2025-03-07T15:17:30.736+0100] {processor.py:208} INFO - Processing /Users/gabi/codes/climate_trace/climate_data_pipeline/dags/climate-data-dag.py took 0.973 seconds
